src/test/scala/pl/project13/scala/akka/raft/CandidateTest.scala

```diff
@@ -72,4 +73,24 @@ class CandidateTest extends RaftSpec with BeforeAndAfterEach
     }
   }
 
+	it should "reject candidate if got RequestVote message with a stale term number" in {
+		candidate.setState(Candidate, data)
+
+		candidate ! RequestVote(Term(1), self, Term(1), 1)
+		fishForMessage(max = 5 seconds) {
```

Comment:
When using `fishForMessage` it's nice to include `fishForMessage(hint = "awaiting DeclineCandidate", ...)`;
gives better error messages.

Test is good though :)